src/app/components/task/DatetimeRespondTask.js

```diff
@@ -203,96 +176,103 @@ class DatetimeRespondTask extends Component {
-                }}
-                renderInput={params => (
-                  <TextField
-                    {...params}
-                    variant="outlined"
+            </Box>
+          </FlexRow>
+        </Box>
+
+        <Box clone mb={2}>
+          <FlexRow >
+            <Box clone flex={10} >
+              <Box clone justifyContent='flex-start' alignItems='center'>
+                <FlexRow
+                  id="task__response-time"
+                >
+                  <TimePicker
+                    id="task__response-time-input"
                     label={
                       <FormattedMessage
-                        id="datetimeRespondTask.timezone"
-                        defaultMessage="Select a timezone"
+                        id="datetimeRespondTask.time"
+                        defaultMessage="Pick a time"
                       />
                     }
+                    onChange={this.handleChangeTime.bind(this)}
+                    value={
+                      this.state.hour && this.state.minute ?
+                        new Date().setHours(this.state.hour, this.state.minute) : null
+                    }
+                    inputVariant="outlined"
+                    InputProps={{
+                      endAdornment: (
+                        <InputAdornment>
+                          <AccessTimeIcon />
+                        </InputAdornment>
+                      ),
+                    }}
+                    variant="inline"
+                    fullWidth
                   />
-                )}
-                fullWidth
-              />
-            </FlexRow>
-          </div>
-        </FlexRow>
-        <div style={styles.error}>
+                  <Box clone ml={2}>
+                    <Autocomplete
+                      className="task__datetime-timezone"
+                      options={
+                        taskTimezones && taskTimezones.length ? taskTimezones : Object.values(timezones)
+                      }
+                      getOptionLabel={option => option.label}
+                      defaultValue={this.state.timezone}
+                      onChange={(event, newValue) => {
+                        this.handleChangeTimezone(newValue);
+                      }}
+                      renderInput={params => (
+                        <TextField
+                          {...params}
+                          variant="outlined"
+                          label={
+                            <FormattedMessage
+                              id="datetimeRespondTask.timezone"
+                              defaultMessage="Select a timezone"
+                            />
+                          }
+                        />
+                      )}
+                      fullWidth
+                    />
+                  </Box>
+                </FlexRow>
+              </Box>
+            </Box>
+          </FlexRow>  
```

Comment:
`271:21  error  Trailing spaces not allowed                      no-trailing-spaces`